.github/workflows/test-exploration.yml

```diff
@@ -0,0 +1,62 @@
+name: Test / Exploration
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  unit:
+    name: Unit Tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+      - run: npm test
+        env:
+          CI: true
```

Comment:
I wonder if we can utilize [composite run steps](https://github.blog/changelog/2020-08-07-github-actions-composite-run-steps/) for this file and the existing workflow?